pom.xml

```diff
@@ -23,6 +23,8 @@
     <properties>
         <osgi.bundle.version>${project.version}</osgi.bundle.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.scm.id>github</project.scm.id>
+        <revision>0.0.1-SNAPSHOT</revision>
```

Comment:
How is this revision different from `${revision}`?

Reply:
This is what sets `${revision}`

Reply:
ah, thx :)